aws/api-servers.tf

```diff
@@ -3,17 +3,19 @@ resource "aws_launch_configuration" "api" {
   image_id = "${lookup(var.amis, var.region)}"
   instance_type = "t2.medium"
   security_groups = ["${aws_security_group.default.id}"]
-  key_name = "${aws_key_pair.deployer.key_name}"
+  key_name = "${var.key_pair_name}"
+/*
```

Comment:
Can we delete these `lifecycle` blocks rather than comment them out?

We can restore them from the commit history if we later find a solution. But for the moment, by being there unused, they're adding more complexity than value.